From c3c089e8a08731aa95b6c11fdd3733a5458b1683 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Thu, 9 Jul 2020 15:34:40 +0700
Subject: [PATCH 338/441] dt-bindings: clock: renesas: cpg-mssr: Document
 r8a774a3 binding

Add binding documentation for the RZ/G2M v3.0 (R8A774A3) Clock Pulse
Generator driver.
Also update name for RZ/G2M v1.3 (R8A774A1) Clock Pulse Generator.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 .../devicetree/bindings/clock/renesas,cpg-mssr.txt         | 14 ++++++++------
 1 file changed, 8 insertions(+), 6 deletions(-)

diff --git a/Documentation/devicetree/bindings/clock/renesas,cpg-mssr.txt b/Documentation/devicetree/bindings/clock/renesas,cpg-mssr.txt
index db1206d..f3e35f2 100644
--- a/Documentation/devicetree/bindings/clock/renesas,cpg-mssr.txt
+++ b/Documentation/devicetree/bindings/clock/renesas,cpg-mssr.txt
@@ -16,7 +16,8 @@ Required Properties:
       - "renesas,r8a7743-cpg-mssr" for the r8a7743 SoC (RZ/G1M)
       - "renesas,r8a7745-cpg-mssr" for the r8a7745 SoC (RZ/G1E)
       - "renesas,r8a77470-cpg-mssr" for the r8a77470 SoC (RZ/G1C)
-      - "renesas,r8a774a1-cpg-mssr" for the r8a774a1 SoC (RZ/G2M)
+      - "renesas,r8a774a1-cpg-mssr" for the r8a774a1 SoC (RZ/G2M v1.3)
+      - "renesas,r8a774a3-cpg-mssr" for the r8a774a3 SoC (RZ/G2M v3.0)
       - "renesas,r8a774b1-cpg-mssr" for the r8a774a1 SoC (RZ/G2N)
       - "renesas,r8a774c0-cpg-mssr" for the r8a774c0 SoC (RZ/G2E)
       - "renesas,r8a774e1-cpg-mssr" for the r8a774e1 SoC (RZ/G2H)
@@ -39,11 +40,12 @@ Required Properties:
   - clocks: References to external parent clocks, one entry for each entry in
     clock-names
   - clock-names: List of external parent clock names. Valid names are:
-      - "extal" (r8a7743, r8a7745, r8a77470, r8a774a1, r8a774b1, r8a774c0,
-		 r8a774e1, r8a7790, r8a7791, r8a7792, r8a7793, r8a7794, r8a7795,
-		 r8a7796, r8a77965, r8a77970, r8a77980, r8a77990, r8a77995)
-      - "extalr" (r8a774a1, r8a774b1, r8a774e1, r8a7795, r8a7796, r8a77965,
-		  r8a77970, r8a77980)
+      - "extal" (r8a7743, r8a7745, r8a77470, r8a774a1, r8a774a3, r8a774b1,
+		 r8a774c0, r8a774e1, r8a7790, r8a7791, r8a7792, r8a7793,
+		 r8a7794, r8a7795, r8a7796, r8a77965, r8a77970, r8a77980,
+		 r8a77990, r8a77995)
+      - "extalr" (r8a774a1, r8a774a3, r8a774b1, r8a774e1, r8a7795, r8a7796,
+		  r8a77965, r8a77970, r8a77980)
       - "usb_extal" (r8a7743, r8a7745, r8a77470, r8a7790, r8a7791, r8a7793,
 		     r8a7794)
 
-- 
2.7.4

